packages/taiko-client/prover/proof_producer/sgx_producer.go

```diff
@@ -117,13 +129,155 @@ func (s *SGXProofProducer) RequestProof(
 	}, nil
 }
 
+func (s *SGXProofProducer) Aggregate(
+	ctx context.Context,
+	items []*ProofWithHeader,
+	requestAt time.Time,
+) (*BatchProofs, error) {
+	log.Info(
+		"Aggregate sgx batch proofs from raiko-host service",
+		"proofs", items,
```

Comment:
```suggestion
		"items", items,
```

they are still not proofs right?

Reply:
They have individual proof in each item but no aggregated proof.